.circleci/config.yml

```diff
@@ -110,10 +113,25 @@ jobs:
           command: PLATFORM=iOS make test
           no_output_timeout: "20m" # 20 minutes
 
-      - store_artifacts:
-          path: /tmp/xcode_raw.log
-      - store_artifacts:
-          path: /tmp/FailureDiffs
+      - restore_cache:
+          name: Restore Bundler
+          keys:
+            - v1-gems-{{ checksum "Gemfile.lock" }}
+
+      - run:
+          name: Install Bundler
+          command: bundle check || bundle install
+          environment:
+            BUNDLE_JOBS: 4
+            BUNDLE_RETRY: 3
+
+      - run:
+          name: Generate Slather coverage report
+          command: bundle exec slather coverage -x --scheme Kickstarter-Framework-iOS ./Kickstarter.xcodeproj
+
+      - run:
+          name: Upload report to codecov
+          command: bash <(curl -s https://codecov.io/bash) -f ./cobertura.xml
```

Comment:
So since we're using the CircleCI Orb for Codecov, we get a sugary syntax for effectively this command that looks like:
```
- codecov/upload:
    file: cobertura.xml
```
You can see how the orb just translates this into a `curl` command.
I wasn't sure why this was uploading without `CODECOV_TOKEN` set on the environment, but it's because this repo is public (which [codecov is fine with](https://docs.codecov.io/docs/about-the-codecov-bash-uploader#upload-token)) 😄

Reply:
oh nice, thanks i'll update that. I should still run the step just before to generate with slather though, right?

Reply:
Yep sounds right!